packages/common/src/__tests__/waku-broadcaster-client.test.ts

```diff
@@ -17,13 +17,13 @@ import { contentTopics } from '../waku/waku-topics.js';
 chai.use(chaiAsPromised);
 const { expect } = chai;
 
-const chain = MOCK_CHAIN_ETHEREUM;
+const chain = { type: 0, id: 42161 } as Chain;
```

Comment:
Going to make the testnet chain / token a constant for all tests to share

Reply:
this can be moved back, there is a test broadcaster on ethereum now.

Reply:
still can do this, but it can be switched back to the original mock-chain here.